src/app/static/src/app/types.ts

```diff
@@ -226,3 +226,8 @@ export interface GenericChartDataset {
         }
     }
 }
+
+export interface DefaultDataSource {
+    datasetId: string | null,
+    showDataPicker: boolean
+}
```

Comment:
Two observations on this:
- `DefaultDataSource` isn't a great name - this isn't a data source (or directly related, really)
- Not sure of value in bundling these two props into a new type - the two fields aren't really attributes of a common entity. Could they just be two individual props?